File: VelibStations_Android/app/src/main/java/com/muhammedsafiulazam/velibstations/feature/stationinfo/event/StationInfoEventType.kt
package com.muhammedsafiulazam.velibstations.feature.stationinfo.event

object StationInfoEventType {
    // Tag.
    private const val TAG = "STATION_INFO_EVENT_TYPE_"

    // Event types.
    const val VIEWMODEL_REQUEST_LOAD_DATA: String = TAG + "VIEWMODEL_REQUEST_LOAD_DATA"
    const val VIEWMODEL_RESPONSE_LOAD_DATA: String = TAG + "VIEWMODEL_RESPONSE_LOAD_DATA"

    const val MODEL_REQUEST_LOAD_DATA: String = TAG + "MODEL_REQUEST_LOAD_DATA"
    const val MODEL_RESPONSE_LOAD_DATA: String = TAG + "MODEL_RESPONSE_LOAD_DATA"
}

File: VelibStations_iOS/VelibStations/Feature/StationInfo/Event/StationInfoEventType.swift
//
//  StationInfoEventType.swift
//  VelibStations
//
//  Created by Safi on 09/10/2019.
//

import Foundation

class StationInfoEventType {
    // Tag.
    private static let TAG: String = "STATION_INFO_EVENT_TYPE_"
    
    // Event types.
    static let VIEWMODEL_REQUEST_LOAD_DATA: String = TAG + "VIEWMODEL_REQUEST_LOAD_DATA"
    static let VIEWMODEL_RESPONSE_LOAD_DATA: String = TAG + "VIEWMODEL_RESPONSE_LOAD_DATA"
    
    static let MODEL_REQUEST_LOAD_DATA: String = TAG + "MODEL_REQUEST_LOAD_DATA"
    static let MODEL_RESPONSE_LOAD_DATA: String = TAG + "MODEL_RESPONSE_LOAD_DATA"
}
